tests/Kernel/StatusMessagesTest.php

```diff
@@ -63,10 +63,10 @@ public function testStatusMessages(array $data): void {
       $heading = self::MESSAGE_TYPES[$type]['heading'];
 
       $wrapper = $crawler->filter('div.ecl-message--' . $modifier);
-      $this->assertCount(1, $wrapper, sprintf('Wrong number of wrappers found for "%s" messages.', $type));
+      $this->assertEquals(1, $wrapper->count(), sprintf('Wrong number of wrappers found for "%s" messages.', $type));
 
       $title = $wrapper->filter('div.ecl-message__title');
-      $this->assertCount(1, $title, sprintf('Wrong number of headings found for "%s" messages.', $type));
+      $this->assertEquals(1, $title->count(), sprintf('Wrong number of headings found for "%s" messages.', $type));
```

Comment:
Ah no sorry! I meant the contrary: to convert all other tests to use `assertCount()`, not to remove it from here.